docs(wildfly): document wildfly 27 and jakarta support (#1424)
camunda/camunda-docs-manual: content/installation/full/jboss/manual.md

---

This document describes the installation of Camunda Platform and its components on a vanilla [Wildfly Application Server](http://www.wildfly.org) or JBoss EAP 7.

{{< note title="Reading this Guide" class="info" >}}
This guide uses a number of variables to denote common path names and constants:
`$WILDFLY_HOME` points to the JBoss EAP/Wildfly application server main directory.

> `$PLATFORM_VERSION` denotes the version of the Camunda Platform you want to install or already have installed, e.g. `7.0.0`.

Author: 👍

{{< /note >}}

> ## Required Setup

Author: 👍

### Wildfly 27+

For Wildfly 27  and above, please download the [Camunda WildFly distribution](https://downloads.camunda.cloud/release/camunda-bpm/wildfly/).

### Wildfly ≤26 / JBoss EAP 7

> For Wildfly ≤26 / JBoss EAP 7, please download the [`camunda-wildfly26-modules`](https://artifacts.camunda.com/artifactory/camunda-bpm/org/camunda/bpm/wildfly/camunda-wildfly26-modules/).

Author: 👍

{{< /note >}}

### Adjust the Configuration

Next, a number of changes need to be performed in the application server's configuration file.
In most cases this is `$WILDFLY_HOME/standalone/configuration/standalone.xml`.

Add the Camunda subsystem as extension:

```xml

> <server xmlns="urn:jboss:domain:20.0">

Reviewer: 👍 Nice catch!

* [How to configure an Oracle database](http://www.ironjacamar.org/doc/userguide/1.0/en-US/html_single/#ex_datasources_oracle)
* [How to configure a MySQL database](http://www.ironjacamar.org/doc/userguide/1.0/en-US/html_single/#ex_datasources_mysql)

## Optional Components

> This section describes how to install optional dependencies. None of these are required to work with the core platform. Before continuing, make sure that the Camunda Platform is already installed according to [this step]({{< relref "#required-setup" >}}) for WildFly / JBoss EAP 7.

Reviewer: 🔧 If we adjust the required setup section name, we need to adjust it here as well.
```suggestion
This section describes how to install optional dependencies. None of these are required to work with the core platform. Before continuing, make sure that the Camunda Platform is already installed according to [this step]({{< relref "#setup" >}}) for WildFly / JBoss EAP 7.
```

```

### Camunda Spin

> The Camunda Spin plugin can be used to extend the engine functionality to de-/serialize object variables from and to JSON and XML. For more information, see the [Spin Reference]({{< ref "/reference/spin/_index.md" >}}).

Reviewer: 🔧 Let's try to use active voice here.
```suggestion
You can use the Camunda Spin plugin to extend the engine functionality to de-/serialize object variables from and to JSON and XML. For more information, see the [Spin Reference]({{< ref "/reference/spin/_index.md" >}}).
```

Author: 👍
